resources/srs/README.md

```diff
@@ -0,0 +1,80 @@
+
+These files are only a portion of the total SRS data that exists for eigenDA. The files here are sufficiently large
+to support the largest permitted blob size of 16MB. This maximum blob size may be increased at some point in the future,
+at which time larger SRS files will be needed.
+
+Note that the G2 point files (`g2.point` and `g2.trailing.point`) are twice the size of the G1 point file because G2 
+points require twice as many bytes to represent as G1 points in the BN254 curve. Each G1 point requires 32 bytes 
+of storage, while each G2 point requires 64 bytes.
+
+## Retrieving SRS Files
+
+These SRS files can be fetched from the AWS S3 bucket at:
+```
+https://srs-mainnet.s3.amazonaws.com/kzg/
+```
+
+### Using the Download Script
+
+The EigenDA repository provides a convenience script located at `tools/download-srs.sh` for downloading and 
+generating SRS file hashes.
+
+Usage:
+```bash
+# Download SRS files for a specified blob size (in bytes)
+./tools/download-srs.sh <size_in_bytes>
+
+# Example: Download SRS files for 16MB blobs
+./tools/download-srs.sh 16777216
+```
+
+The script will:
+1. Download the appropriate portions of g1.point and g2.point files
+2. Calculate the correct byte range for g2.trailing.point
+3. Save all files to a "srs-files" directory
+4. Generate SHA256 hashes for verification
+5. Create a hash file with labeled hashes for each downloaded file
+
+## SRS Verification and Alternative Retrieval Method
+
+For users who need to verify the integrity of SRS files, please refer to the
+[SRS Utilities README](/tools/srs-utils/README.md) for detailed instructions. This tool provides:
+
+1. Methods to extract and verify G1 and G2 points from the original Perpetual Powers of Tau challenge file
+2. Verification procedures to ensure the correctness of the SRS points based on approaches used by the Ethereum 
+   Foundation's KZG ceremony
+3. Ability to parse the full 8GB SRS files from the original challenge file, which can then be manually truncated 
+   to smaller sizes as needed
+
+The SRS utilities provide an alternative approach to obtaining SRS files by downloading the original challenge file 
+directly from the Ethereum Foundation's trusted setup, extracting the points, and verifying their integrity.
+
+## Security Considerations
+
+Using the correct SRS files is essential for the proper functioning of any software interacting with EigenDA. 
+If a user has incorrect or tampered SRS files, the following would occur:
+
+1. **Verification failures**: The user would be unable to successfully verify KZG commitments and proofs, making it 
+   impossible to validate blob data from the network.
+
+2. **Network incompatibility**: A node using incorrect SRS files would be unable to meaningfully interact with the 
+   EigenDA network, as it would consistently fail to verify certificates from honest nodes.
```

Comment:
eigenda nodes don't verify certs right afaiu? what "node" are you referring to?

Reply:
I was using "node" in a more general sense, which is confusing. 

I've rewritten this section of the doc in [b0bc39e7](https://github.com/Layr-Labs/eigenda/pull/1458/commits/b0bc39e7da8e6a87e3c783eb2698f8e0f828a54c)